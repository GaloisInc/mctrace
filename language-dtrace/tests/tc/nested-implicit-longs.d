proc:::exec_success {
  // This is testing whether or not the type checker is able to
  // traverse a nested chain of assignments
  l1 = l2 = l3 = 5l;
}
