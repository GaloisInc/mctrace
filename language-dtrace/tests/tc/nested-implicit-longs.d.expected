TCExpected { globals = [ ("l1", BVType 64)
                     , ("l2", BVType 64)
                     , ("l3", BVType 64)
                     ]
         }
